src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.UnmodifiableObservableList;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.person.Entry;
+import seedu.address.model.person.FloatingTask;
+import seedu.address.model.person.Title;
+import seedu.address.model.person.UniquePersonList.DuplicateTaskException;
+import seedu.address.model.person.UniquePersonList.PersonNotFoundException;
+
+/*
+ * Edit a task's content.
+ */
+public class EditCommand extends Command {
+
+    public static final String COMMAND_WORD = "edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits tasks details. " + "Parameters: TASK_ID"
+            + "[TITLE]" + "Example: " + COMMAND_WORD + " 2 Buy bread";
+
+    public static final String MESSAGE_SUCCESS = "Edited entry: %1$s";
```

Comment:
`1$` [indicates the position of the argument being passed in](https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html#syntax), I guess it can be simplified too, was following the convention from the existing code base :v